components/TaskList.tsx

```diff
@@ -1,9 +1,15 @@
 import { Link, List, Tag } from "lbh-frontend-react/components";
 import querystring from "querystring";
-import { nullAsUndefined } from "null-as-undefined";
 import PropTypes from "prop-types";
 import React from "react";
 
+export enum Status {
```

Comment:
```suggestion
export enum TaskListStatus {
```

To avoid name clashes.